pcsx2/x86/newVif_Dynarec.cpp

```diff
@@ -263,6 +258,26 @@ void VifUnpackSSE_Dynarec::ModUnpack(int upknum, bool PostOp)
 	}
```

Comment:
on a line a couple up it says "pxFailRel(fmt::format("Vpu/Vif - Invalid Unpack! [%d]", upknum).c_str());" but it never gives the the value of upknum. 
![crash](https://user-images.githubusercontent.com/8851653/235825232-a0c1b6db-6efe-47cf-9a8d-57d874759375.jpg)

Reply:
the value of it is gonna always be 11